vmdb/app/views/ops/_ldap_domain_show.html.haml

```diff
@@ -0,0 +1,101 @@
+#main_div
+  = render(:partial => "layouts/flash_msg")
+  %fieldset
+    %p.legend= _("Basic Information")
+    %table.style1
+      %tr
+        %td.key= _("Name")
+        %td= @selected_ld.name
+  %fieldset
+    %p.legend= _("LDAP Settings")
+    %table.style1
+      %tr
+        %td.key= _("User Type")
+        %td.wide
+          - case @selected_ld.user_type
```

Comment:
can't we here avoid the assignment to `user_type` and just leave the case as an expression?